.github/workflows/helm.yml

```diff
@@ -86,10 +86,10 @@ jobs:
         run: |
           eval $(minikube -p minikube docker-env)
           ./gradlew \
-            :polaris-quarkus-server:assemble \
-            :polaris-quarkus-server:quarkusAppPartsBuild --rerun \
-            :polaris-quarkus-admin:assemble \
-            :polaris-quarkus-admin:quarkusAppPartsBuild --rerun \
+            :polaris-runtime-server:assemble \
```

Comment:
As stated in the ML by @snazy I'm not sure we need the word `runtime` in the artifact names (the directory name is fine though).

Reply:
Thanks for your input ! Much appreciated ! Let me do the change.

Reply:
+1

Reply:
+1

Reply:
I'm now using `polaris-server` and `polaris-admin` as project names.